acasclient/acasclient.py

```diff
@@ -244,9 +245,10 @@ def getSession(self):
         session = requests.Session()
         resp = session.post("{}/login".format(self.url),
                             headers={'Content-Type': 'application/json'},
-                            data=json.dumps(data))
+                            data=json.dumps(data),
+                            allow_redirects=False)
         resp.raise_for_status()
-        if 'location' in resp.headers and resp.headers.location == "/login":
+        if resp.status_code == 302 and 'location' in resp.headers and resp.headers.get('location') == "/login":
```

Comment:
Nit: Just to future proof this a little in case we end up with a /acas prefix to our urls, it might be good to set this to:
```
resp.headers.get('location').endswith('/login')
```

That being said, we should probably be giving a 401 when hitting login via api so that probably a better fix down the line.

Reply:
Yeah, going to leave this as-is. I'm not sure how it'd handle things like `/login?redirect=foo` and I agree a status code would be a better fix, but considering that out of scope.